x-pack/plugins/apm/common/apm_index_pattern.ts

```diff
@@ -0,0 +1,7 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export const APM_STATIC_INDEX_PATTERN_ID = 'apm_static_index_pattern_id';
```

Comment:
This file already exists in https://github.com/elastic/kibana/blob/master/src/plugins/apm_oss/common/index_pattern_constants.ts. Can we avoid duplicating it?